src/commons/BrowserUtils.ts

```diff
@@ -103,7 +103,7 @@ export namespace BrowserUtils {
    */
   export function click(selector: string): void {
     Reporter.debug(`Click an element '${selector}'`);
-    isVisible(selector);
+    isDisplayed(selector);
```

Comment:
we might add waitForEnabled before click too

Reply:
done